jigsaw-game/common/__init__.py

```diff
@@ -2,18 +2,38 @@
 """Core Image Manipulation functionality"""
 
 from math import sqrt
-
-from PIL.Image import Image, open
+import requests
+from PIL import Image
+import random
+#from PIL.Image import Image, open
 
 
 def get_image(
     filename: str, max_dimensions: tuple[int, int], num_of_tiles: int
 ) -> Image:
     """This could be extended to by addeding filters, and from other sources"""
-    with open(filename) as image:
-        image.thumbnail(max_dimensions)
-        num_of_tiles = int(sqrt(num_of_tiles))
-        width = image.width // num_of_tiles
-        height = image.height // num_of_tiles
-        image = image.crop((0, 0, num_of_tiles * width, num_of_tiles * height))
-    return image
+##    with open(filename) as image:
+##        image.thumbnail(max_dimensions)
+##        num_of_tiles = int(sqrt(num_of_tiles))
+##        width = image.width // num_of_tiles
+##        height = image.height // num_of_tiles
+##        image = image.crop((0, 0, num_of_tiles * width, num_of_tiles * height))
+    PEXELS_API_KEY = '1cK1A6oZ1wPOB49B2qMohsQCWp2aIEU8FlRzQIt9bL3AzXzjle5fmRsh'
```

Comment:
We can move this to a `.env` file if you want to create one of those and then import that here as [such](https://stackoverflow.com/questions/40216311/reading-in-environment-variables-from-an-environment-file).